### PR TITLE
Removed gpuOff.

### PR DESCRIPTION
**Pull request by emilwestergren**
@sebastianbaginski peer review

### CODE REVIEW

**Comment by emilwestergren** [comment]
@davidpiuva

**Comment by emilwestergren** [comment]
@erikhagglund ???????????????

File: test.sh [hunk 1]
```diff
@@ -23,7 +23,7 @@ esac
 FEATURES=""
 if [[ $1 == "nogpu" ]]
 then
-	FEATURES=$FEATURES" -DgpuOff"
+	FEATURES=$FEATURES
```
Comment (erikhagglund) [inline test.sh]: seems a bit redundant... :)

Comment (emilwestergren) [inline test.sh]: Me remove will

**Comment by emilwestergren** [comment]
Updated

File: test.sh [hunk 2]
```diff
@@ -21,10 +21,6 @@ case "$1" in
 		;;
 esac
 FEATURES=""
```
Comment (erikhagglund) [inline test.sh]: ```FEATURES``` is no longer used

**Comment by emilwestergren** [comment]
Updated

**Pull request merged**